packages/desktop-client/src/components/accounts/Account.js

```diff
@@ -519,9 +535,9 @@ function SelectedTransactionsButton({
   onUnlink,
   onCreateRule,
   onScheduleAction,
+  history,
 }) {
   let selectedItems = useSelectedItems();
-  let history = useHistory();
```

Comment:
Previously `SelectedTransactionsButton` was the only function that used the `history.push` element. Now I need to use it in other functions as well. So I define it one level up then share it with all the needed functions as a prop rather than defining it multiple times.